capstone/src/main/java/com/google/univiz/SuggestionData.java

```diff
@@ -1,4 +1,18 @@
 package com.google.univiz;
 
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import javax.annotation.Nullable;
+
 /** SuggestionData represents the college data specific to the Search feature */
-public class SuggestionData {}
+@GenerateTypeAdapter
+@AutoValue
+abstract class SuggestionData {
+  @SerializedName("school.name")
+  @Nullable
+  abstract String name();
+
+  @SerializedName("id")
+  abstract int id();
```

Comment:
Wasn't this going to be CollegeId?

Reply:
Yes-I have a note about this in the description!

Reply:
If you see a way to address the problem in the PR description, we can talk offline and I will incorporate it into the PR